data_prep.R

```diff
@@ -32,21 +30,18 @@ lakes_wq_dat<-read_xlsx('inputs/Herrera Lakes All.xlsx',
          Month=month(DateTime),
          WaterYear=ifelse(Month>=10,Year+1,Year),
          FakeDate=as.Date(paste(2000,Month,day(DateTime),sep='-')),
-         WY_FakeDate=as.Date(if_else(Month>=10,FakeDate-years(1),FakeDate))) %>%
-  mutate(parameter=ifelse(parameter=='Water transparency','Secchi Depth',parameter))
-
-lakes_wq_dat["parameter"][lakes_wq_dat["parameter"] == "Temperature, water"] <- "Water Temperature (°C)"
+         WY_FakeDate=as.Date(if_else(Month>=10,FakeDate-years(1),FakeDate)))
```

Comment:
Somewhere in the code there is the following error: 
Error: first mandatory vector should be positive: year or year+fraction
       second mandatory vector should be numerical: measured data (ties allowed)
       third optional vector should be positive integer: season, month, site or a unique code for season and site
       fourth optional vector should be numerical: covariable
       all with the same length, at least 4

This seems to happen whenever water quality trends are calculated with not enough data. This is one of the lines I could find that referenced Date, but I'm not sure this is the actual issue.